doc/tutorials/pixel_data/compressing.rst

```diff
@@ -1,230 +1,425 @@
-========================
-Compressing *Pixel Data*
-========================
+====================================================
+*Pixel Data* - Part 3: Compression and decompression
+====================================================
 
 .. currentmodule:: pydicom
 
-This tutorial is about compressing a dataset's *Pixel Data* and covers
-
-* An introduction to compression
-* Using data compressed by third-party packages
-* Compressing data using *pydicom*
-
-It's assumed that you're already familiar with the :doc:`dataset basics
-<../dataset_basics>`.
 
+In part 1 of this tutorial you learnt how to :doc:`access the pixel data
+</tutorials/pixel_data/introduction>` as either the raw :class:`bytes` or a NumPy
+:class:`~numpy.ndarray` and in part 2 you learnt how to :doc:`create new pixel data
+</tutorials/pixel_data/creation>` and add it to a :class:`~pydicom.dataset.Dataset`.
+In this final part you'll learn how to compress and decompress datasets containing
+*Pixel Data*.
 
 **Prerequisites**
 
-This tutorial uses packages in addition to *pydicom* that are not installed
-by default, but are required for *RLE Lossless* and *JPEG-LS Near-lossless*
-compression of *Pixel Data*. For more information on what packages are
-available to compress a given transfer syntax see the :ref:`image compression
-guide <guide_compression_supported>`.
-
 Installing using pip:
 
 .. code-block:: bash
 
-    python -m pip install -U pydicom numpy pylibjpeg pyjpegls
+    python -m pip install -U pydicom numpy matplotlib pylibjpeg[all] pyjpegls
```

Comment:
I haven't seen matplotlib used here, maybe I missed it

Reply:
Removed